clientd3d/d3drender_world.c

```diff
@@ -1888,17 +1930,11 @@ void D3DRenderLMapPostWallAdd(WallData *pWall, d3d_render_pool_new *pPool, unsig
          normal.z = vec0.y * vec1.x - vec0.x * vec1.y;
          normal.y = vec0.x * vec1.z - vec0.z * vec1.x;
 
-         custom_xyz lightVec;
-         float cosAngle;
-
-         lightVec.x = pDLightCache->dLights[numLights].xyz.x - xyz[0].x;
-         lightVec.y = pDLightCache->dLights[numLights].xyz.y - xyz[0].y;
-         lightVec.z = pDLightCache->dLights[numLights].xyz.z - xyz[0].z;
-
-         cosAngle = lightVec.x * normal.x + lightVec.y * normal.y + lightVec.z * normal.z;
-
-         if (cosAngle <= 0)
-            continue;
+         // NOTE: Removed cosAngle backface culling check for walls.
```

Comment:
Documentation for code that doesn't exist seems a little odd.  How about just saying why we're not doing backface culling here (beyond "causing issues")?